documentation/asciidoc/topics/yaml/basic_auth.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 stringData:
-  username: operator <1>
-  password: dIRs5cAAsHIeeRIL <2>
+  username: operator # User that can access the {brandname} cluster.
```

Comment:
This has to be the `operator` credential that is generated or specified in the `<cluster-name>-generated-operator-secret`